cluster/log-dump/log-dump.sh

```diff
@@ -415,7 +416,7 @@ function find_non_logexported_nodes() {
 function dump_nodes_with_logexporter() {
   if [[ -n "${use_custom_instance_list}" ]]; then
     echo "Dumping logs for nodes provided by log_dump_custom_get_instances() function"
-    NODE_NAMES=( $(log_dump_custom_get_instances node) )
+    while IFS='' read -r line; do NODE_NAMES+=("$line"); done < <(log_dump_custom_get_instances node)
```

Comment:
```
In ./cluster/log-dump/log-dump.sh line 418:
    NODE_NAMES=( $(log_dump_custom_get_instances node) )
                 ^-- SC2207: Prefer mapfile or read -a to split command output (or quote to avoid splitting).
```